packages/flutter/lib/src/material/theme.dart

```diff
@@ -143,7 +144,12 @@ class Theme extends StatelessWidget {
       theme: this,
       child: IconTheme(
         data: data.iconTheme,
-        child: child,
+        child: CupertinoTheme(
+          data: MaterialBasedCupertinoThemeData(
```

Comment:
Would be great to have a comment that explains the whole "material" aspect of MaterialBasedCupertinoThemeData here

Reply:
SG, done